src/BlaspExpressionService.php

```diff
@@ -84,6 +112,13 @@ abstract class BlaspExpressionService
      */
     protected array $falsePositives;
 
+    /**
+     * Static cache for language-specific configurations
+     *
+     * @var array
+     */
+    protected array $cache = [];
```

Comment:
Another feature I wanted to add was a cache but to cache the entire blasp object. we can keep this for now I just wanted to share.

Reply:
I would like to know more about your idea. However, I will remove this cache in my next pull request.